changelogs/unreleased/fix-bootstrap-with-er-readonly.md

```diff
@@ -0,0 +1,4 @@
+## bugfix/replication
```

Comment:
Lets add `Part of #7737` and mention the ticket ID in the changelog. (Or `Closes` if you will decided to fix the ticket completely.)

Reply:
Sure. I missed your ticket somehow.

Speaking of the second problem (about some of the tests failing), I could reproduce it as well, but couldn't find the reason yet. It's rather hard to debug.